hack/functests.sh

```diff
@@ -45,6 +45,9 @@ function functest() {
         echo "Will skip test asserting the cluster is in dual-stack mode."
         extra_args="${extra_args} -skip-dual-stack-test"
     fi
+
+    extra_args="${extra_args}  -conn-check-ipv4-address=${conn_check_ipv4_address} -conn-check-ipv6-address=${conn_check_ipv6_address} -conn-check-dns=${conn_check_dns}"
```

Comment:
why not to combine this with line 43?

Reply:
no good reason.
There use to be an `if -fi` there so it had to be separate.When  I removed the `if-fi` I didn't combined them.

Done.